dnsimple/domains_test.go

```diff
@@ -48,12 +48,16 @@ func TestDomainsService_ListDomains(t *testing.T) {
 		t.Errorf("Domains.ListDomains() expected to return %v contacts, got %v", want, got)
 	}
 
-	if want, got := int64(1), domains[0].ID; want != got {
+	if want, got := int64(181984), domains[0].ID; want != got {
 		t.Fatalf("Domains.ListDomains() returned ID expected to be `%v`, got `%v`", want, got)
 	}
 	if want, got := "example-alpha.com", domains[0].Name; want != got {
 		t.Fatalf("Domains.ListDomains() returned Name expected to be `%v`, got `%v`", want, got)
 	}
+
+	if want, got := "2021-06-05", domains[0].ExpiresOn; want != got {
```

Comment:
@onlyhavecans @DXTimer @duduribeiro I see two different attributes referenced in the same spec (aka the message doesn't match the code). Is that on purpose?

Reply:
No, my fault!. should be expiresOn.